lib/game.ml

```diff
@@ -65,10 +65,10 @@ module Lvl1 = struct
     (e, t, f, f, f, t, f, f, f) row
   ) grid
 
-  type start = <world:world; player:player_start; lvl:lvl; init:player_turn >
+  type 'count start = <count: 'count; world:world; player:player_start; lvl:lvl; init:player_turn >
```

Comment:
I am thinking of using a more diegetic name: either stamina (as a field in the player state), or score.